### PR TITLE
fix: account menu makeover when account name is larger

### PR DESCRIPTION
**Pull request by pavanjoshi914**
before: 

image

after:
image

### CODE REVIEW

**Review by reneaaron — COMMENTED** [review]
Great catch!

File: src/app/components/AccountMenu/index.tsx [hunk 1]
```diff
@@ -103,7 +103,7 @@ function AccountMenu({ showOptions = true }: Props) {
                         url={authAccount.avatarUrl}
                       />
                     </div>
-                    <div className="flex flex-col">
+                    <div className="flex flex-col max-w-40">
```
[the remark this answers is not in the document]
Comment (pavanjoshi914) [reply src/app/components/AccountMenu/index.tsx:106]: resolved

**Review by reneaaron — APPROVED** [review]
utACK

**Pull request merged**